starter/script.js

```diff
@@ -1 +1,56 @@
-// put code here!
+//=== Program wide functions ===
+var badResponse = function() {
```

Comment:
Very nice! Keeping that code DRY by condensing repeated lines of code in a function.